src/synapse/README.rst

```diff
@@ -0,0 +1,5 @@
+Microsoft Azure CLI 'synapse' Extension
```

Comment:
Remove this file since we already have `README.md`

Reply:
deleted

---
In reply to: [386412229](https://github.com/Azure/azure-cli-extensions/pull/1339#discussion_r386412229) [](ancestors = 386412229)